CHANGELOG.md

```diff
@@ -1,50 +1,952 @@
+
+##### Added
+- PHP 7 support
+- [a basic knowledgebase](https://github.com/shopsys/shopsys/blob/master/docs/index.md)
+    - installation guide
+    - guidelines for contributions
+    - cookbooks
+    - articles on automated testing
+
+##### Changed
+- update to Symfony 3
+- PSR-2 compliance
+- English as a main language
+    - language of first front-end domain
+    - language of administration
+    - all translatable message sources in English
+
+##### Deleted
+- separation of HTTP smoke test module into a component:
+    - https://github.com/shopsys/http-smoke-testing/
+- separation of product feed modules into plugins:
+    - https://github.com/shopsys/plugin-interface/
+    - https://github.com/shopsys/product-feed-interface/
+    - https://github.com/shopsys/product-feed-heureka/
+    - https://github.com/shopsys/product-feed-heureka-delivery/
+    - https://github.com/shopsys/product-feed-zbozi/
+
+#### 6.0.0-alpha - 2016-11-09
+- developed since 2014-03-31
+- used only as internal platform for e-commerce projects of Shopsys Agency
+- released only internally
+
+##### Added
+- product catalogue
+- registered customers
+- basic orders management
+- back-end administration
+- front-end fulltext search
+- front-end product filtering
+- 3-step ordering process
+- products variants
+- simple promo codes
+- product feeds for product aggregators
+- basic CMS
+- multiple administrators
+- support for several currencies
+- support for several languages
+- support for several domains
+- full friendly URL for main entities
+- customizable SEO attributes for main entities
+
+#### [shopsys/coding-standards]
+##### [4.0](https://github.com/shopsys/coding-standards/compare/v3.1.1...v4.0.0) - 2018-01-27
+###### Added
+- composer script `run-all-checks` for easier testing of the package (@TomasVotruba)
+
+###### Changed
+- `OrmJoinColumnRequireNullableFixer` marked as *risky* (@sustmi)
+- [#11](https://github.com/shopsys/coding-standards/pull/11) dropped support of PHP 7.0 
+- [#12](https://github.com/shopsys/coding-standards/pull/12/) [EasyCodingStandard](https://github.com/Symplify/EasyCodingStandard) is now used (@TomasVotruba)
+    - the tool encapsulates PHP-CS-Fixer and PHP_CodeSniffer 
+    - rules configuration is now unified in single file - [`easy-coding-standard.neon`](./easy-coding-standard.neon)
+    - the option `ignore-whitespace` for rules checking method and class length is not available anymore
+        - the limits were increased to 550 (class length) and 60 (method length)
+    
+###### Removed
+- PHP Mess Detector (@TomasVotruba)
+- line length sniff (@TomasVotruba)
+
+##### [3.1.1](https://github.com/shopsys/coding-standards/compare/v3.1.0...v3.1.1) - 2017-10-31
+###### Fixed
+- enabled custom fixers
+
+##### [3.1.0](https://github.com/shopsys/coding-standards/compare/v3.0.2...v3.1.0) - 2017-10-12
+###### Added
+- This changelog 
+- [Description of used coding standards rules](./docs/description-of-used-coding-standards-rules.md) 
```

Comment:
404

Reply:
this acutally does not exist anymore, i made change so it actually looks into the coding-standards package, but it does not exist